src/Operation/Count.php

```diff
@@ -162,6 +162,39 @@ public function execute(Server $server)
         return (integer) $result->n;
     }
 
+    /**
+     * Explain the operation.
+     *
+     * @see Explainable::explain()
+     * @param Server $server
+     * @param $command
+     * @param array $options
+     * @return array|object
+     * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     */
+    public function explain($server, $command, $options = [])
+    {
+        if ( ! isset($options['verbosity'])) {
+            $options['verbosity'] = 'allPlansExecution';
```

Comment:
While I doubt this will break sharding (like MapReduce recently), it may be safer to simply omit this option unless the user specifies a value. I believe the server will enforce this default if `verbosity` is absent in the `explain` command document.

Side note: we can also create constants for the three known modes on the Explain class, as you did with Watch and the lookup modes. Users don't _need_ to use those constants, but we can use them in tests and it's a nice reminder for those browsing the code.